site/src/testHelpers/entities.ts

```diff
@@ -215,6 +215,7 @@ export const MockProxyLatencies: Record<string, ProxyLatencyReport> = {
 
 export const MockBuildInfo: TypesGen.BuildInfoResponse = {
 	agent_api_version: "1.0",
+	provisioner_api_version: "1.1",
```

Comment:
Since current is `2.2` do we want that as the default instead?

Reply:
It's 1.1 isn't it?
https://github.com/coder/coder/blob/7518aac879cdcda88aa7f6af6c6841ea6f280ee4/provisionerd/proto/version.go#L5-L14

Reply:
`agent_api_version` is the one that's currently 2.2

Reply:
Ah agent api is something else